src/Type/IntrospectionProvider.php

```diff
@@ -215,14 +218,13 @@ protected function registerIntrospectionTypes()
                     'name'            => GraphQL::TYPE_INTROSPECTION,
                     'isIntrospection' => true,
                     'description'     =>
-                        'The fundamental unit of any GraphQL Schema is the type. There are ' .
-                        'many kinds of types in GraphQL as represented by the `__TypeKind` enum.' .
-                        '\n\nDepending on the kind of a type, certain fields describe ' .
```

Comment:
Took a while to find that this line was not in double-quotes, so linebreaks didn't work as expected. 😐